scripts/signature.py

```diff
@@ -93,10 +98,56 @@ def diff(args):
         """
         )
 
-    return exit_status
+    return delta, exit_status
+
+
+def diff(args):
+    """Diffs Velox function signatures."""
+    return diff_signatures(args)[1]
+
+
+def dir_path(string):
+    if os.path.isdir(string):
+        return string
+    else:
+        raise NotADirectoryError(string)
+
+
+def bias(args):
+    """Writes out newly added functions to be biased to a file.
+    The file, named bias_functions, is written to the specified output directory.
+    Contents are of the form : `fn_name1=10,fn_name2=10`.
+    Returns 1 if it detects incompatible changes.
+    """
+
+    delta, status = diff_signatures(args)
+
+    # Return if the signature check call flags incompatible changes.
+    if status:
+        return status
+
+    if not len(delta):
+        print(f"{bcolors.BOLD} No changes detected: Nothing to do!")
+        return 0
+
+    function_set = set()
+    for items in delta.values():
+        for item in items:
+            function_set.add(item.get_root_key())
+
+    print(f"{bcolors.BOLD}Functions to be biased: {function_set}")
+    output_dir = args.output_dir
+
+    function_bias_output = f"{output_dir}/bias_functions"
+
+    if len(function_set):
+        with open(function_bias_output, "w") as f:
+            print(f"{'=10,'.join(sorted(function_set)) + '=10'}", file=f, end="")
```

Comment:
should we take this "10" as an argparse parameter - and maybe set its default value to 10?